python/py-pew/files/pew37

```diff
@@ -0,0 +1 @@
+bin/pew-3.7
```

Comment:
Is this the actual main script or it's still a symlink? If the latter is the case, it would be better to point to the actual script - a subtle improvement for the startup time.

Reply:
Yes, please do.

BTW, you may want to use variables like https://github.com/macports/macports-ports/blob/master/python/py-pip/files/pip37, so that users with custom `frameworks_dir` (unlikely but possible) can use it.